tools/make-compile-commands

```diff
@@ -44,7 +44,6 @@ for line in output.splitlines():
     if not match:
         continue
 
-    cc = match.group(1)
```

Comment:
I think I had that there as documentation.

Can you add a comment for this one, as per the `# match group 4` line just below?

Reply:
Good idea, done.